modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -1,50 +1,35 @@
 // Module included in the following assemblies:
 // TODO
 // * networking/TBD
+// * networking/load-balancing-openstack.adoc
 // For thinking and reviewing, adding to networking/load-balancing-openstack.adoc
 
 [id="nw-osp-configuring-external-load-balancer_{context}"]
 = Configuring an external load balancer
 
-You can configure a {product-title} cluster on {rh-openstack-first} to use an external load balancer.
+You can configure a {product-title} cluster on {rh-openstack-first} to use an external load balancer in place of the default load balancer.
 
 // Maybe an About mod in support
 
 .Prerequisites
 
-* On your load balancer, ports 6443, 443, and 80 must be available to any users of your system.
-* On your load balancer, port 22623, which is used to serve ignition start-up configurations to nodes, is not exposed outside of the cluster.
+* On your load balancer, TCP over ports 6443, 443, and 80 must be available to any users of your system.
 
-.Procedure
+** The {product-title} API uses port 6443. 
```

Comment:
We can be a little more specific. I think something like this could be a good compromise:
Load balance the api port, 6443, between each of the control plane nodes. Load balance the application ports, 443 and 80, between all of the compute nodes.

Reply:
L16 is not what I thought I committed, so this all reads a bit strangely. 😬 I think I deleted the good line of text and left the old one. Whoops. 

Happy with your suggestion as 18-20, though, if that seems copacetic to you.